apps/console/src/extensions/components/users/pages/users.tsx

```diff
@@ -445,27 +449,118 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
         />
     );
 
+    const addUserDropdownTrigger: ReactElement = (
+        <>
```

Comment:
We can remove the fragment component (`<></>`), as we already have a parent component.